.gitignore

```diff
@@ -111,6 +111,16 @@ Thumbs.db
 #############################
 doc/source/savefig/
 
+# Azure Pipeline generated files #
+##################################
+venv/*
+
+# github actions generated files #
+##################################
+log
+warnings
+_configtest.gcno
+pypy3/*
```

Comment:
`pypy3` comes from [`.github/workflows/build_test.yml`](https://github.com/numpy/numpy/blob/master/.github/workflows/build_test.yml#L187) and could be refactored to move outside the source directory.

`venv` comes from [`azure-pipelines.yml`](https://github.com/numpy/numpy/blob/master/azure-pipelines.yml#L49) and could also be refactored.

`*.gcno` is valid to add: it refers to the GCC `-ftest-coverage` option.

I am not sure where `log` and `warnings` come from.